Incorrect version range in `engines.node` field in package.json

I'm trying to install version 4.2.1 via pnpm (thanks for releasing this new version by the way!) - but am getting this error:

```
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

This error happened while installing a direct dependency of /Users/dan/Source/westfield/hello-world-component/packages/infrastructure

Your Node version is incompatible with "cdk-nextjs-standalone@4.2.1".

Expected version: >= >=18.18.0
Got: v22.14.0

This is happening because the package's manifest has an engines.node field specified.
To fix this issue, install the required Node version.
```

I am using the [engine-strict](https://pnpm.io/npmrc#engine-strict) option in pnpm, and the `engines.node` field is `>= >=18.18.0` (with 2 `>=`s)

It looks like this problem was introduced in https://github.com/jetbridge/cdk-nextjs/pull/245 with `>=` being added to line 38 of `.projenrc.ts`:

```
minNodeVersion: '>=18.18.0'
```

--- a/.projenrc.ts
+++ b/.projenrc.ts
@@ -18,6 +18,8 @@ const commonTscOptions: TypeScriptCompilerOptions = {
   // esModuleInterop: true, // why doesn't this work?
 };
 
+const minNodeVersion = '18.18.0';
+
 const project = new awscdk.AwsCdkConstructLibrary({
   // repository config
   author: 'JetBridge',
@@ -35,7 +37,8 @@ const project = new awscdk.AwsCdkConstructLibrary({
   packageName: 'cdk-nextjs-standalone',
   majorVersion: 4,
   // prerelease: 'beta',
-  minNodeVersion: '>=18.18.0',
+  minNodeVersion,
+  workflowNodeVersion: `>=${minNodeVersion}`,
   description: 'Deploy a NextJS app to AWS using CDK and OpenNext.',
   keywords: [
     'nextjs',

--- a/package.json
+++ b/package.json
@@ -98,7 +98,7 @@
     "standalone"
   ],
   "engines": {
-    "node": ">= >=18.18.0"
+    "node": ">= 18.18.0"
   },
   "main": "lib/index.js",
   "license": "Apache-2.0",

